.kokoro/continuous/node10/system-test.cfg

```diff
@@ -1,7 +0,0 @@
-# Download resources for system tests (service account key, etc.)
-gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/google-cloud-nodejs"
-
-env_vars: {
```

Comment:
If we're going to get rid of our standard system tests, we need to put an exception in sloth.

Reply:
+1 - Should be an easy change in sloth when we're ready